sdk_lib/sdk_container_common.sh

```diff
@@ -11,6 +11,11 @@ sdk_container_common_versionfile="sdk_container/.repo/manifests/version.txt"
 sdk_container_common_registry="ghcr.io/flatcar-linux"
 sdk_container_common_env_file="sdk_container/.sdkenv"
 
+docker="docker"
+if docker --version | grep -q -i podman; then
```

Comment:
I think we could extend the test in case the user has not aliased `docker` with `podman` ?

Reply:
An alias from `.bashrc` doesn't matter for scripts

Reply:
Ok - but then, this command will just fail if there is no `docker` on the system.

On my machine:
```
./run_sdk_container -t
sdk_lib/sdk_container_common.sh: line 15: docker: command not found

###### Writing versionfile 'sdk_container/.repo/manifests/version.txt' to SDK '3087.0.0', OS '3066.0.0+56-ga4864fd9'. ######
./run_sdk_container: line 85: docker: command not found

###### Creating a new container 'flatcar-sdk-all-3087.0.0_os-alpha-3066.0.0-56-ga4864fd9' ######
./run_sdk_container: line 107: docker: command not found
```

Reply:
Misunderstood it, will change to check for the podman binary and use it preferably

Reply:
Done